plugins/woocommerce/tests/php/src/Internal/Admin/Schedulers/OrdersSchedulerTest.php

```
@@ -443,6 +443,137 @@ public function test_is_scheduled_import_enabled_prefers_new_option(): void {
+		$status_before = $wpdb->get_var(
+			$wpdb->prepare(
+				"SELECT status FROM {$wpdb->prefix}wc_order_stats WHERE order_id = %d",
+				$order->get_id()
+			)
+		);
+		$this->assertSame( 'wc-completed', $status_before );
+
+		// Track whether the hook fires and whether delete_post fires.
+		$hook_fired    = false;
+		$delete_fired  = false;
+		$import_result = null;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$import_result'. (undefined)

(UnusedLocalVariable)